reference/deprecation-notices.md

```diff
@@ -3,6 +3,18 @@
 
 This document contains a list of deprecation notices for Anbox Cloud and its components.
 
+## Kernel 6.8 and earlier
+*Deprecated in 1.25.0* ; *Removal in 1.27.0*
+
+The support for kernel 6.8 and earlier is deprecated as of 1.25.0. Starting from 1.27.0, Anbox Cloud will run only on kernels with version later than 6.8.
+
+## LXD 5.0
+*Deprecated in 1.25.0* ; *Removal in 1.27.0*
+
+The support for LXD 5.0 snap is deprecated as of 1.25.0. Upgrade your deployment to use LXD 5.21 instead.
+
+If you wish to continue with LXD 5.0, you must change the value of `volume.zfs.block_mode` to `true` for your LXD storage pool.
```

Comment:
I think we had a misunderstanding for this: users should not set this value to true. It _should_ be reverted automatically by AMS or the appliance anyway, but this line should be removed.